client/src/document/writer-toolbar/spy.tsx

```diff
@@ -42,16 +46,17 @@ export default function DocumentSpy({ onMessage }) {
       mounted = false;
       wssRef.current && wssRef.current.close();
     };
-  }, [onMessage]);
+  }, [documentURL, onDocumentUpdate]);
 
   if (connected === null) {
     return null;
   }
 
   return (
     <div
-      id="document-spy"
-      className={connected ? "ws-connected" : "ws-not-connected"}
+      className={`document-spy ${
+        connected ? "ws-connected" : "ws-not-connected"
+      }`}
```

Comment:
While you're in there, do you think you could add a `title` attribute to this DOM node. That way, when you hover over it wondering "What the heck is this?!" it could show a tooltip like "Connected to the document build watcher" or "Not connected to the document build watcher".

Reply:
I'm not sure what the title could get there that we don't already get from the contained text nodes. Imo 

> "Connected to the document build watcher" or "Not connected to the document build watcher".

would be repetition here. Or are there accessibility reasons to prefer it?